packages/typespec-test/test/widget_dpg/generated/typespec-ts/review/widget_dpg.api.md

```diff
@@ -38,13 +35,13 @@ export interface ListWidgetsOptions extends OperationOptions {
 
 // @public (undocumented)
 export interface UpdateWidgetOptions extends OperationOptions {
-    color?: ColorType;
```

Comment:
The ColorType is made by TCGC, now they prefer to model it as string. See comments https://github.com/Azure/autorest.typescript/pull/2029#discussion_r1330336733.

Personally I prefer not to give a name for it. Because we don't know the name and it may conflict with customer's own.